packages/react/src/internals/context.tsx

```diff
@@ -26,6 +26,9 @@ export type TRPCContextState<TRouter extends AnyRouter> = {
   client: TRPCClient<TRouter>;
   isPrepass: boolean;
 
+  /**
+   * @link https://react-query.tanstack.com/guides/prefetching
```

Comment:
Hmm what's the difference between this and `prefetchQuery` below? Based on the link here, is it the same? If so maybe this one should be deprecated too.

Reply:
It's a weird one but quite nice DX-detail.

`prefetchQuery` **never** `throw`s so it's very useful for optimistic prefetch where you don't care about caring about error state so useful for things like what I outlined in #914